web/packages/core/src/current-script.ts

```diff
@@ -24,3 +24,34 @@ try {
 } catch (_e) {
     console.warn("Unable to get currentScript URL");
 }
+
+/**
+ *
+ * Obtain the origin src content according to the running environment of the \<script\> node
+ *
+ * @param script \<script\> node instance
+ *
+ * @returns \<script\> node origin src
+ */
+function getScriptOriginalSrc(script: HTMLScriptElement): string {
+    const scriptUrl = script.src;
+    const scriptUrlPolyfill = script.getAttribute("ruffle-src-polyfill");
+    if (!scriptUrlPolyfill) {
+        return scriptUrl;
+    }
+    // Reset webkit mask url should be safe
+    if ("webkit-masked-url://hidden/" === scriptUrl) {
```

Comment:
It feels slightly unnecessary to have the early return above when this condition can just be:

`if (scriptUrlPolyfill && "webkit-masked-url://hidden/" === scriptUrl)`